1-exercises/B-while-loop/exercise.js

```diff
@@ -1,12 +1,24 @@
 /*
-    while loops can be useful when you want to execute some code as long as some condition is true.    
+    while loops can be useful when you want to execute
+    some code as long as some condition is true.    
 
-    Using a while loop, complete the function below so it logs (using console.log) the first n even numbers as a comma-seperated string.
+    Using a while loop, complete the function below so 
+    it logs (using console.log) the first n even numbers as a comma-seperated string.
     The list of numbers should start with 0. n is being passed in as a parameter.
 */
 
 function evenNumbers(n) {
     // TODO
+let i=0;
+let evenNum=[];
+while(evenNum.length < n){
+    evenNum.push(i);
+    
+    i+=2;
+
+}
+console.log(evenNum.join()); // join("") will return a string with no commas wheras join("-") will add hyphen. leaving it empty will insert commas in array
+
 }
 
 evenNumbers(3); // should output 0,2,4
```

Comment:
Its generally convention to indent the lines inside a function, so that its easy to see what is inside vs, outside. There are extensions that you can install in vscode such as 'prettier' that automatically do this for you when you save!